### PR TITLE
KEYCLOAK-8010 Improve handling of Conditions SAML tag

### PR DESCRIPTION
**Pull request by hmlnarik**

### CODE REVIEW

**Review by pskopek — CHANGES_REQUESTED** [review]
Only minor issue (see other comment).
Otherwise it looks good to me.

File: testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/broker/KcSamlIdPInitiatedSsoTest.java [hunk 1]
```diff
@@ -38,10 +45,7 @@
 import org.openqa.selenium.support.ui.ExpectedCondition;
 import org.openqa.selenium.support.ui.WebDriverWait;
 
-import static org.hamcrest.Matchers.containsInAnyOrder;
-import static org.hamcrest.Matchers.containsString;
-import static org.hamcrest.Matchers.hasSize;
-import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.*;
```
[the remark this answers is not in the document]
Comment (hmlnarik) [reply testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/broker/KcSamlIdPInitiatedSsoTest.java]: You're indeed right, fixing in next commit. Thanks for spotting this!

**Comment by hmlnarik** [comment]
@pskopek Done, could you please re-review?

**Review by pskopek — APPROVED** [review]
Looks good!

**Review by mposolda — APPROVED** [review]

**Pull request merged**